### PR TITLE
Assignment 2 submission

### PR DESCRIPTION
**Pull request by Swaroopaamruthwar**

### CODE REVIEW

File: src/Assignment-2/assignment2.html [hunk 1]
```diff
@@ -10,6 +10,71 @@
...
+			</section>
+		     <aside id="right-sidebar-menu">
+					<ul>
+						<li>section 1</li>
+						<li>section 2</li>
+						<li>section 3</li>
+						<li>section 4</li>
+						<li>section 5</li>
+						<li>section 6</li>
+						<li>section 7</li>
```
Comment (nikhilkuruba) [inline src/Assignment-2/assignment2.html:66]: Code is fine, but I think this list should contain only 6 list items

**Review by nikhilkuruba — APPROVED** [review]

**Review by nikhilkuruba — APPROVED** [review]

File: src/Assignment-2/assignment2.html (hunk 1 above)
Comment (Swaroopaamruthwar) [reply src/Assignment-2/assignment2.html:66]: ok i will do changes to it

Comment (Swaroopaamruthwar) [reply src/Assignment-2/assignment2.html:66]: ok..

**Comment by SuneetKumar19** [comment]
Good

**Pull request merged**